go/api/sorted_set_commands.go

```diff
@@ -0,0 +1,31 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+/** SetCommands supports commands and transactions for the "Sorted Set Commands" group for standalone and cluster clients.
+ *
+ * See [valkey.io] for details.
+ *
+ * [valkey.io]: https://valkey.io/commands/?group=sorted-set#sorted-set
+ */
+type SortedSetCommands interface {
+	// Increments the score of member in the sorted set stored at key by increment.
+	// If member does not exist in the sorted set, it is added with increment as its score.
+	// If key does not exist, a new sorted set with the specified member as its sole member
+	// is created.
+	//
+	// see [valkey zincrby] for details.
```

Comment:
```suggestion
	// See [valkey.io] for details.
```

Reply:
Why is this an issue?  The two tags match and I think the [valkey zincrby] tag provides more context to the user.

Reply:
Existing godocs all adopted the valkey.io tag so I think for now we should keep them consistent

Reply:
I will change it back for now, although I think we should revisit this later as we should not be encouraging cargo culting.